websites/D/DMOJ/presence.ts

```diff
@@ -0,0 +1,191 @@
+const presence = new Presence({
+    clientId: "754070047193956492",
+});
+
+const browsingStamp = Math.floor(Date.now() / 1000);

```

Comment:
merge consts

Reply:
How would i go about doing this? (I'm new to typescript/javascript so i don't know much about the technicalities)

Would it be something like

```ts
const presence = new Presence({
    clientId: "754070047193956492",
    browsingStamp: Math.floor(Date.now() / 1000);
});
```

Reply:
```
const presence = new Presence({
    clientId: "754070047193956492",
}),
 browsingStamp = Math.floor(Date.now() / 1000);
```